smithy-typescript-ssdk-libs/server-node/src/node.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ *  Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License").
+ *  You may not use this file except in compliance with the License.
+ *  A copy of the License is located at
+ *
+ *   http://aws.amazon.com/apache2.0
+ *
+ *  or in the "license" file accompanying this file. This file is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *  express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+
+import { HttpRequest, HttpResponse } from "@aws-sdk/protocol-http";
+import { HeaderBag, QueryParameterBag } from "@aws-sdk/types";
+import { IncomingHttpHeaders, IncomingMessage, ServerResponse } from "http";
+
+function convertHeaders(headers: IncomingHttpHeaders): HeaderBag {
+  // TODO make this proper
+  return Object.fromEntries(Object.entries(headers).filter((x) => x)) as HeaderBag;
+}
+
+function converQueryString(qs: URLSearchParams): QueryParameterBag {
```

Comment:
URLSearchParams, URL don't need to be imported?
Also missing `t` in function name.